02_activities/assignments/assignment.sh

```diff
@@ -8,17 +8,22 @@ set -x
 # > Add your code below each comment to complete the tasks
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
-
+mkdir mydirectorie
+cd mydirectorie
+mkdir dir1 dir2 dir3 dir4 dir5
 # 2. List the contents of the parent directory to verify the presence of the 5 directories
-
+ls 
 # 3. Create 5 text files in dir2 named file1, file2, file3, file4, and file5
-
+cd dir2
+touch file1 file2 file3 file4 file5
 # 4. Append the words "Hello world" to file3
-
+echo  "hello world"  >> file3 
 # 5. Verify that file3 contains the words "hello world" by printing the contents of the file in the terminal
-
+cat file3
 # 6. Delete file4
-
+rm file4
 # 7. Delete directories dir4 and dir5 including all their contents (if any)
-
+cd .. 
+rm -r dir4 dir5
```

Comment:
missing .txt extensions

For better practice, remove cd dir2 in question 3 and define the relative path when operating on file or directories (eg. cat dir2/file3.txt)